Templates/BlobTrigger-CSharp/sample.dat

```diff
@@ -0,0 +1,3 @@
+{
+    "id":"foobar123"
+}
```

Comment:
Yes. Maybe my sample data can be more clear it is a sample.

Sent from Outlook Mobile




On Mon, Mar 7, 2016 at 10:22 PM -0800, "Fabio Cavalcante" <notifications@github.com> wrote:





> @@ -0,0 +1,3 @@
> +{
> +    "id":"foobar123"
> +}

Is this supposed to be here?

---
Reply to this email directly or view it on GitHub:
https://github.com/Azure/azure-webjobs-sdk-templates/pull/7/files#r55320767